ui/src/app/views/workspace/component.ts

```diff
@@ -17,10 +41,20 @@ export class WorkspaceComponent implements OnInit {
   public static DEFAULT_WORKSPACE_NS = 'defaultNamespace';
   public static DEFAULT_WORKSPACE_NAME = 'defaultWorkspace';
   public static DEFAULT_WORKSPACE_ID = '1';
+  private cohortNameFilter = new CohortNameFilter();
+  private cohortDescriptionFilter = new CohortDescriptionFilter();
+  private notebookNameFilter = new NotebookNameFilter();
+  private notebookDescriptionFilter = new NotebookDescriptionFilter();
 
   repositories: Repository[] = [];
   user: User;  // to detect if logged in
   cohortList: Cohort[] = [];
+  notebookList = [new Notebook('Notebook 1',
+                    'This is the user defined description for notebook 1',
```

Comment:
Please include "TODO" here. I'm being paraoid about making sure we can sweep up all test/fake data easily before launch, since I have launched standin values to prod before.